h2o-core/src/main/java/hex/Model.java

```diff
@@ -104,6 +104,14 @@ public double defaultThreshold() {
 
   public final boolean isSupervised() { return _output.isSupervised(); }
 
+  public boolean havePojo() {
+    return ModelBuilder.havePojo(_parms.algoName().toLowerCase());
```

Comment:
Very minor suggestion: perhaps you can move the toLowerCase to ensureBuilderIndex method?

Reply:
👍